### PR TITLE
Skip client provisioning on end to end tests

### PR DESCRIPTION
**Pull request by elyezer**
If the clients support is not available on the configuration, skip the end to
end client provisioning.

### CODE REVIEW

**Comment by sthirugn** [comment]
ACK

File: tests/foreman/endtoend/utils.py [hunk 1]
```diff
@@ -21,6 +22,8 @@ def client_provisioning(
             is available.
         :param package_name: Name of the package to be installed on the client.
         """
+        if not setting_is_set('clients'):
```
Comment (oshtaier) [inline tests/foreman/endtoend/utils.py]: Why not to use `skip_if_not_set`? Or if we need specific function for that purpose, why do we put it in `robottelo/decorators.py` module as it is not real decorator?

Comment (elyezer) [inline tests/foreman/endtoend/utils.py]: skip_if_not_set is not a function but a decorator. It have the same functionality of is_bz_bug_open for example, it is a function that allows skip_if_not_set do its job and it is also made available public to that module for usage outside the module.

Comment (oshtaier) [inline tests/foreman/endtoend/utils.py]: Thanks

**Comment by oshtaier** [comment]
ACK

**Comment by abalakh** [comment]
ACK

**Pull request merged**